sequencing-server/src/lib/codegen/CommandTypeCodegen.ts

```diff
@@ -121,23 +137,48 @@ function ${fswCommandName}(...args: [{ ${argsWithType.map(arg => arg.name + ': '
   };
 }
 
+function generateHwCommandCode(hwCommand: ampcs.HwCommand): { value: string; interfaces: string } {
+  const needsUnderscore =
+    /^\d/.test(hwCommand.stem) ||
+    reservedWords.check(hwCommand.stem) ||
+    typescriptReservedWords.includes(hwCommand.stem);
+
+  const hwCommandName = (needsUnderscore ? '_' : '') + hwCommand.stem;
+
+  const doc = generateDoc(hwCommand);
+  const value =
+    `${doc}` +
+    `\nconst ${hwCommandName}: ${hwCommandName} = HardwareStem.new({` +
+    `\n\tstem: '${hwCommand.stem}'` +
+    `\n})`;
+
+  const interfaces =
+    `${doc}` + `\ninterface ${hwCommandName} extends HardwareStem {}\nconst ${hwCommandName}: ${hwCommandName}`;
+  return {
+    value,
+    interfaces,
+  };
+}
+
 /**
  * Creates a jsdoc style doc for the given command. Right now it just includes the args as
  * parameters.
  *
  * @param fswCommand The command we're generating documentation for.
```

Comment:
Super minor but the comment here could be updated to match the new name of the argument.

Reply:
Thanks for catching that, I updated the parameter